### ISSUE TITLE
OpenRocket Crashes

### ISSUE THREAD

**Issue by roguextech** [openrocket/openrocket#987]
I double-clicked the desktop shortcut, and a couple of seconds later, what you see in the screenshot is what happened.

I have attached the ORK file, user thrust curves and screenshot. The attached file is a 7zip file renamed to the .gif extension. I will be in and out this evening for about the next 4 hours. I am assuming that ya'll may want more info, I will get back with you.

JP

![jpjohnston-openrocket-BUG-20210801 7z](https://user-images.githubusercontent.com/55761077/127786724-ea4c9f61-afcd-483b-a435-63d0bf1e0763.gif)

**Comment by SiboVG** [comment]
Whilst I do not know the root cause of the issue, if you want to still keep using OR, you can temporarily move your user-defined thrust curves to somewhere else. I was able to recreate your issue when I moved the thrust curves that you included to my user thrust curves directory.

**Comment by roguextech** [comment]
I figured out which file is causing the issue. RATT.eng is the culprit...

**Comment by SiboVG** [comment]
Yup, indeed, you can also see inside the RATT.eng-file that there are indeed two thrust-values for a single time point, which is the thing that OR complains about...
![image](https://user-images.githubusercontent.com/11031519/132993290-1fc07d71-edac-41fc-9b8c-0a9af27dcaec.png)

The thing is, the error lies within the RATT.eng file and not really within OR itself; there shouldn't really be 2 or more thrust-values for a single time point.

What I can however do is show a pop-up window saying something in the line of "Thrust curve 'RATT.eng' has two thrust values for single time point, time[3]=0.278207, thrust=240.028; time[4]=0.278207, thrust=229.017. 'RATT.eng' will thus be ignored for importing the thrust curves. You can try replacing/deleting the thrust curve or manually editing it so that each time point has only one thrust value."
